api/v1beta1/machine_types.go

```diff
@@ -50,6 +50,9 @@ const (
 	// to pause reconciliation of deletion. These hooks will prevent removal of
 	// an instance from an infrastructure provider until all are removed.
 	PreTerminateDeleteHookAnnotationPrefix = "pre-terminate.delete.hook.machine.cluster.x-k8s.io"
+
+	// MachineCertificatesExpiryDateAnnotation annotation specifies the expiry date of the machine certificates.
```

Comment:
Let's document that:
- this annotation can be used by MHC to trigger machine rotation before cert expiration
- this annotation can be set in two places, Machine having priority on the boostrap config object